<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalConfigDialog</class>
 <widget class="QDialog" name="GlobalConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>731</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>网络</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>540</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>例子：关节1对应&quot;&lt;J1&gt;&quot;,关节2对应&quot;&lt;J2&gt;&quot;...依次类推，默认为:&quot;[&lt;J1&gt;,&lt;J2&gt;,&lt;J3&gt;,&lt;J4&gt;,&lt;J5&gt;,&lt;J6&gt;]&quot;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>386</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="network_port_lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>621</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>协议</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="communication_protocol"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>271</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="read_cmd">
           <property name="text">
            <string>读取指令</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="read_feedback">
           <property name="text">
            <string>读取反馈</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>目标IP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="read_worldcmd">
         <property name="text">
          <string>世界指令</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="tcp_port"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="tcp_ip_addr"/>
       </item>
       <item row="3" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="auto_connect_checkBox">
           <property name="text">
            <string>自动重连</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="auto_connect_lineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="tcp_checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>主动通讯</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="udp_checkBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>被动通讯</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>基本</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>215</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>恢复默认参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="open_last_project">
       <property name="text">
        <string>默认打开上一次模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="debug_log_checkBox">
       <property name="text">
        <string>调试日志输出</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>刷新周期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="refreshCycleLineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>轨迹最大点数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_point_num_lineedit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>轨迹记录配置</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>开启轨迹记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_en"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>红色轨迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_red"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>黄色轨迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_yellow"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>蓝色轨迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_blue"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>绿色轨迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_green"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
